tests/Tests/E2e/ExampleE2eTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
this script looks like a partial duplicate of tests/Tests/E2e/TestOpenemrPage/ExampleE2eTest.php . Is there a reason for this (ie. migrating test location maybe?)?

Reply:
Yes that script should not be there. I had some issues running e2e issues and was messing with files, this must have survived the cleanup... I'll remove this file.